tests/test_edges.py

```diff
@@ -71,7 +71,24 @@ def test_save_bulk (self, client, test_client):
         openreview.tools.post_bulk_edges(client, edges)
         posted_edges = list(openreview.tools.iterget_edges(client, invitation=inv1.id))
         assert len(edges) == len(posted_edges)
-
+    
+    def test_rename_edges(self, client):
+        guest = openreview.Client()
+        to_profile = guest.register_user(email = 'lbahy@mail.com', first = 'Nadia', last = 'LBahy', password = '1234')
```

Comment:
This line is failing when all tests run with:
```js
{'name': 'Error', 'message': 'User not confirmed. Please click on "Didn\'t receive email confirmation?" to complete the registration.', 'status': 400}
```

This happened because test_client.py already registers this email. One solution is to change the email and the name to something else. Only changing the email would not be sufficient because then, the new profile would be assigned ~Nadia_LBahy2 if all tests are run and ~Nadia_LBahy1 if only test_edges.py is run.

Reply:
Thank you for pointing this out, I'll fix that now!